### ISSUE TITLE
[SI-ERROR-TRUE]  2.4.4 Link Purpose (In Context) - The link text is not sufficient (Issue content type)

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#1035]
## Problem Description per Siteimprove

The link text is generic text that does not identify the link purpose. The link does not seem to have any programmatically determined link context that can aid in identifying the purpose either. Link texts should be written so they make sense out of context. Generic texts such as “Click here” and “More” give no indication as to the destination of the links.

## Page Description

An instance of the Issue content type. 

## What we're doing

A bunch of articles with 'Read More' link

## Solution

Possibly concatenate 'Read More' + 'about' + first few words of article title

## Note

This is very similar to issue #1036 where the links are non-distinguishable

**Comment by tovstiadi** [comment]
From a test with a screen reader user on July 1: another ambiguous link text is "A to Z" - visually that is next to CU so it is clear that it is "CU A to Z" but not obvious to a screen reader user.

**Comment by tovstiadi** [comment]
From a test with a screen reader user on July 1: we did find something curious here: content MORE accessible to a screen reader user than the sighted user. On an article with tags, the screen reads the icons as “tags” and “categories” (=meaningful clear labels) while sighted user gets two icons that look like a folder and a tag - not as transparent.
